client/changes/bug_6797_add-dependency-on-twisted

```diff
@@ -0,0 +1,2 @@
+  o Add dependency on Twisted. Closes #6797.
+  o Remove unneeded dependency on oauth.
```

Comment:
if I understand it right, we're just moving it to common (since both server and client depend on it).
Can you clarify this for the changelog? (might point that it is an indirect dep because u1db doesn't declare it)